examples/platforms/nrf528xx/src/radio.c

```diff
@@ -41,6 +41,8 @@
 #include <stdint.h>
 #include <string.h>
 
+#include <openthread/link.h>
```

Comment:
This is fine. Just wondering what definition/API from `link.h` is used?

Reply:
It's `OT_US_PER_TEN_SYMBOLS` defined in `link.h`. It's currently used in both simulation and nrf528xx platforms.